deployment/service.tf

```diff
@@ -0,0 +1,107 @@
+#
+# ECS Resources
+#
+
+# ECS cluster is only a name that ECS machines may join
+resource "aws_ecs_cluster" "gtsite" {
+
+  lifecycle {
+    create_before_destroy = true
+  }
+
+  name = "${var.stack_name}"
+}
+
+# Template for container definition, allows us to inject environment
+data "template_file" "ecs_gtsite_task" {
+  template = "${file("${path.module}/containers.json")}"
+
+  vars {
+    service_image            = "${var.service_image}"
+    static_image             = "${var.static_image}"
+  }
+}
+
+# Allows resource sharing among multiple containers
+resource "aws_ecs_task_definition" "gtsite" {
+  family                = "${var.stack_name}"
+  container_definitions = "${data.template_file.ecs_gtsite_task.rendered}"
+}
+
+# Defines running an ECS task as a service
+resource "aws_ecs_service" "gtsite" {
+  name                               = "${var.stack_name}"
+  cluster                            = "${aws_ecs_cluster.gtsite.id}"
+  task_definition                    = "${aws_ecs_task_definition.gtsite.family}:${aws_ecs_task_definition.gtsite.revision}"
+  desired_count                      = "${var.desired_instance_count}"
+  iam_role                           = "${var.ecs_service_role}"
+  deployment_minimum_healthy_percent = "0" # allow old services to be torn down
+  deployment_maximum_percent         = "100"
+
+  load_balancer {
+    elb_name       = "${aws_elb.gtsite.name}"
+    container_name = "geotrellis-site-static"
+    container_port = 8080
+  }
+}
+
+# Load balance among all running containers
+resource "aws_elb" "gtsite" {
```

Comment:
I think that we should try to use the new ALBs here vs. the previous generation ELBs. It involves a few more resources, but gives us greater flexibility on port mapping and HTTP/2.

An example with ALB use is in https://github.com/azavea/raster-foundry-deployment/blob/develop/terraform/app.tf, but I am working on a PR that is going to tweak the way it works a bit today or tomorrow.